thunder.install

```diff
@@ -249,3 +279,30 @@ function thunder_update_8308() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Install optional modules for users that already use them.
+ */
+function thunder_update_8309() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+
+  if (\Drupal::moduleHandler()->moduleExists('content_moderation')) {
+    /** @var \Drupal\Core\Extension\ModuleInstallerInterface $moduleInstaller */
+    $moduleInstaller = \Drupal::service('module_installer');
+    $moduleInstaller->install(['scheduler_content_moderation_integration', 'config_update']);
+
+    _thunder_bare_install('thunder_workflow');
```

Comment:
I'd be tempted to inline this method so we only have to rebuild the container once if we do an "install" - in fact we could do it so that we only have to write to core.extension once too.

I think not adding a method to do a "bare" install is a good idea - since hopefully we'll never have to do this again.